ui/analyse/src/study/studyCtrl.ts

```diff
@@ -492,8 +499,9 @@ export default class StudyCtrl {
       this.vm.mode.sticky = false;
       if (!this.vm.behind) this.vm.behind = 1;
       this.vm.chapterId = id;
-      componentCallbacks(id);
+      if (this.relay) this.relay.chatCtrl.chapterId = undefined;
```

Comment:
What is this line about? I'm not a fan that it makes studyCtrl interfer with the relay chat implementation details, and I couldn't see a difference when I removed it.

Reply:
> I set relayChat.chapterId to undefined in that case to force relayChat to redraw from the local mainline until the real data shows up. otherwise there is an ugly delay before the live board updates to the new game and that's really disconcerting when you're clicking different boards in rapid succession